### PR TITLE
Fetch samples for table from API

### PR DESCRIPTION
**Pull request by lvreynoso**
### Description

Incorporates the api endpoint for fetching samples into the samples table. The data lives in the data layer for easy switching.

#### Issue
[ch66711](https://app.clubhouse.io/genepi/stories/space/66711)

### Test plan

On local server, navigate to Data. They should load automatically.

### CODE REVIEW

File: src/ts/src/data/samples/index.tsx [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
+//     },
+//     {
+//         privateId: "0865-0004KGK00-001",
+//         publicId: "0909EEEE-55-33",
+//         uploadDate: "2/1/2021",
+//         collectionDate: "12/12/2020",
+//         collectionLocation: "Santa Clara County",
+//         gisaid: "Accepted",
+//     },
+// ];
```
Comment (phoenixAja) [inline src/ts/src/data/samples/index.tsx:43]: i think we can take this out now

File: third-party/sqlalchemy-enum-tables/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta:__legacy__"
```
Comment (phoenixAja) [inline third-party/sqlalchemy-enum-tables/pyproject.toml:3]: did you mean to add this file?

**Review by phoenixAja — COMMENTED** [review]
exciting!! just a few comments, plus those linting issues about types

File: third-party/sqlalchemy-enum-tables/pyproject.toml (hunk 2 above)
Comment (lvreynoso) [reply third-party/sqlalchemy-enum-tables/pyproject.toml:3]: no, I have no idea what this is lol

**Pull request merged**